### PR TITLE
Fixed file share live tests

### PR DESCRIPTION
**Pull request by sima-zhu**
Added another fix of not adding null folder whenever running the tests.

### CODE REVIEW

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy [hunk 1]
```diff
@@ -255,13 +259,13 @@ class FileSASTests extends APISpec {
...
 
-    def "ShareSAS network test identifier permissions create delete"() {
+    def "ShareSAS network identifier permissions"() {
         setup:
         ShareSignedIdentifier identifier = new ShareSignedIdentifier()
             .setId("0000")
             .setAccessPolicy(new ShareAccessPolicy().setPermissions("rcwdl")
                 .setExpiresOn(getUTCNow().plusDays(1)))
-
+        primaryShareClient.create()
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy]: Need to update playback record, this introduces an additional network call,

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy [hunk 2]
```diff
@@ -201,6 +204,7 @@ class FileSASTests extends APISpec {
     def "FileSAS network test upload fails"() {
         setup:
         String data = "test"
+        primaryShareClient.create()
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy]: Need to update playback record, this introduces an additional network call,

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy [hunk 3]
```diff
@@ -141,6 +143,7 @@ class FileSASTests extends APISpec {
     def "FileSAS network test download upload"() {
         setup:
         String data = "test"
+        primaryShareClient.create()
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy]: Need to update playback record, this introduces an additional network call,

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy [hunk 4]
```diff
@@ -24,11 +24,13 @@ class FileSASTests extends APISpec {
     private ShareFileClient primaryFileClient
     private ShareClient primaryShareClient
     private ShareServiceClient primaryFileServiceClient
+    private String shareName
 
-    private String shareName = "sharename"
     private String filePath = "filename"
 
     def setup() {
+        shareName = testResourceName.randomName(methodName, 60)
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy]: Need to update all playback records in this file as this adds a new recording value that will be expected.

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy (hunk 2 above)
Comment (sima-zhu) [reply sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy]: Updated

**Pull request merged**